src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/LanguageServices/SemanticsFactsService/AbstractSemanticFactsService.cs

```diff
@@ -58,7 +58,7 @@ internal abstract class AbstractSemanticFactsService
             IEnumerable<string> usedNames, CancellationToken cancellationToken)
         {
             var container = containerOpt ?? location.AncestorsAndSelf().FirstOrDefault(
-                a => SyntaxFacts.IsExecutableBlock(a) || SyntaxFacts.IsMethodBody(a));
+                a => SyntaxFacts.IsExecutableBlock(a) || SyntaxFacts.IsParameterList(a) || SyntaxFacts.IsMethodBody(a));
```

Comment:
```
                a => SyntaxFacts.IsParameterScope(a) || SyntaxFacts.IsMethodBody(a));
```

Reply:
IsParameterScope isn't something SyntaxFacts should know about.  SyntaxFacts is just a way to structurally ask what type of node something is, in a way that is agnostic to VB/C#.  i.e. if both VB/C# have an InvocationExpressionSyntax, then SyntaxFacts can be used with IsInvocationExpression to ask that question in a uniform way, without worrying about lang. 

It's specifically for syntactic similarity questions, not for semantic questions (like scopes).  Thanks!

Reply:
@CyrusNajmabadi Ok, understood.